lib/6to5/generation/whitespace.js

```diff
@@ -11,15 +11,17 @@ Whitespace.prototype.getNewlinesBefore = function (node) {
   var startToken;
   var endToken;
   var tokens = this.tokens;
+  var token;
+
+  for (var i = 0; i < tokens.length; i++) {
```

Comment:
I don't think they're caching on purpose. They need to know `length` to build an array from `arguments` so it's silly not to use it in the loop if we have it anyway.